config/config.go

```diff
@@ -93,6 +93,10 @@ const (
 	FeatureLogView FeatureName = "logs-tab"
 )
 
+const (
+	IstioClusterTag string = "istio.cluster_id"
+)
```

Comment:
Unless this can be configured I wouldn't keep this in `config` otherwise it could be confusing that there's no config value for this. Maybe move it to `tracing`?

Reply:
Changed the location and rebased. Now passes CI!